app/api/vector.py

```
@@ -0,0 +1,181 @@
+    try:
+        # 벡터 저장소 기본 정보
+        store_stats = vector_store.get_store_stats()
+        
+        # DB 기본 통계
+        active_products_stmt = select(func.count(DBProduct.id)).where(DBProduct.status == "ACTIVE")
+        active_result = await db.execute(active_products_stmt)
+        active_product_count = active_result.scalar() or 0
+        
+        db_embeddings_stmt = select(func.count(DBProductEmbedding.id))
+        db_result = await db.execute(db_embeddings_stmt)
+        db_embedding_count = db_result.scalar() or 0
```

Ruff (0.11.9)

Local variable `db_embedding_count` is assigned to but never used

Remove assignment to unused variable `db_embedding_count`

(F841)